code/modules/reagents/chemistry/items.dm

```diff
@@ -0,0 +1,145 @@
+/obj/item/pHbooklet
+	name = "pH indicator booklet"
+	desc = "A booklet containing paper soaked in universal indicator."
+	icon_state = "pHbooklet"
+	icon = 'icons/obj/chemical.dmi'
+	item_flags = NOBLUDGEON
+	var/numberOfPages = 50
+	resistance_flags = FLAMMABLE
+	w_class = WEIGHT_CLASS_TINY
```

Comment:
```suggestion
	item_flags = NOBLUDGEON
	resistance_flags = FLAMMABLE
	w_class = WEIGHT_CLASS_TINY
	var/numberOfPages = 50
```
Try to put the new (non-redefined) variables at the bottom, for proper organization.
If you could use snake case, that'd keep things more standardized as well.

Reply:
Done